projects/canopy/src/lib/card/card-hero-img/card-hero-img.component.scss

```diff
@@ -0,0 +1,249 @@
+@import '../../../styles/mixins';
+
+.lg-card-hero-image__content {
+  min-width: var(--card-hero-image-width);
+  width: 100%;
+  height: var(--card-hero-image-height);
+  background-repeat: no-repeat;
+  background-position: center center;
+  background-size: cover;
+}
+
+.lg-card-hero-image__src-content {
+  width: 100%;
+  height: 100%;
```

Comment:
I've added the ``height: 100%`` for cases where they use a ``imageSrc`` and have a orientation of 'horizontal'. If I get rid of ``height: 100%`` , in orientation of 'horizontal', it will not fill the container. That is the only reason that I left it there, but if that is the behaviour we want then I can gladly remove it. :) 
<img width="977" alt="Screenshot 2023-08-03 at 13 42 36" src="https://github.com/Legal-and-General/canopy/assets/68600084/3d629ca4-28c0-4858-a4ae-eebbd9625cde">

Reply:
Ah I see :) yes that makes sense